### PR TITLE
test(scorecard): multi-namespace scorecard test for 3.0

### PR DESCRIPTION
**Pull request by mwangggg**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits): `git commit -S -m "YOUR_COMMIT_MESSAGE"`
_______________________________________________

Fixes: #539 

- adds multi-namespace scorecard test for Cryostat 3.0
- amends all the scorecard tests to use operatorv1beta2 resources instead of operatorv1beta1
- removes mention of clustercryostats in bundle & config files

### CODE REVIEW

File: internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml [hunk 1]
```diff
@@ -150,3 +150,12 @@ rules:
...
   - create
+  - delete
+- apiGroups:
+  - operator.cryostat.io
+  resources:
+  - cryostats
+  verbs:
+  - create
+  - delete
+  - get
```
Comment (tthvo) [inline internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml:161]: There is already a rule to create Cryostat above?

Comment (mwangggg) [reply internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml:161]: not in the ClusterRole

Comment (mwangggg) [reply internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml:161]: Otherwise I can't create a Cryostat resource in the targetNamespace

Comment (tthvo) [reply internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml:161]: Ahh okay thanks! I missed that haha :))

**Review by ebaron — APPROVED** [review]
Looks good to me! Thanks @mwangggg!

**Pull request merged**